cspell.config.yaml

```diff
@@ -29,3 +29,4 @@ ignorePaths:
   - 'docs'
   - 'ThirdParty'
   - 'History.txt'
+  - 'CMakeLists.txt'
```

Comment:
Why was this change needed?

Reply:
Running it locally I got this result:
```
# spellCheck
(node:22730) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
 1/38 ./CHANGELOG.md 281.05ms
 2/38 ./CMakeLists.txt 99.14ms X
/Users/skptak/repos/libraries/SigV4-for-AWS-IoT-embedded-sdk/CMakeLists.txt:35:53     - Unknown word (DSIGV)      -- PROJECT_NAME} PRIVATE -DSIGV4_DO_NOT_USE_CUSTOM_CONFIG
	 Suggestions: [sigv, SIGV, SDIV, DSIZE, DESIGN]
```

I would much rather add `DSIGV` to the [.cSpellWords.txt](https://github.com/aws/SigV4-for-AWS-IoT-embedded-sdk/blob/main/.github/.cSpellWords.txt) than to exclude this file from all spell checks.

Reply:
I agree with @Skptak. We should not remove CMakeLists.txt since there are only few violations which we need to add to cSpellWords.txt.

Reply:
seems fine for me, let me change it